web_drop_target/static/src/js/web_drop_target.js

```diff
@@ -106,7 +108,30 @@ odoo.define('web_drop_target', function(require) {
                     );
                 }
             });
-        }
+        },
+
+        _add_overlay() {
```

Comment:
ES6 syntax is forbidden if you don't want to turn developers crazy, as you can see with @simahawk  in https://github.com/OCA/web/issues/980#issuecomment-405177950 :stuck_out_tongue_winking_eye: 

PhantomJS doesn't support it, and minified sources with broken syntax kill the whole JS silently.

Reply:
Upss... sorry! I didn't know about that. 😅